plugins/org.jkiss.dbeaver.data.gis/src/org/jkiss/dbeaver/data/gis/handlers/WKGUtils.java

```diff
@@ -67,4 +74,94 @@ public static DBGeometry parseWKB(String hexString) throws DBCException {
         throw new DBCException("Invalid geometry object");
     }
 
+    public static boolean isCurve(@Nullable Object value) {
+        return value instanceof CircularString
+            || value instanceof CompoundCurve
+            || value instanceof CurvePolygon
+            || value instanceof MultiCurve
+            || value instanceof MultiSurface;
+    }
+
+    @NotNull
+    public static Object linearize(@NotNull Geometry value) {
+        return linearize(value, 1 / 32.0);
```

Comment:
Why 32? Can we add a comment or make a constant?

Reply:
The `ST_CurveToLine` function that converts curved geometry to line geometry uses this value to produce 32 segments per quadrant, so I took it as a reference. I'll add a comment.